src/api/helpers/create-an-application/index.test.ts

```diff
@@ -25,58 +35,103 @@ describe('helpers/create-an-application', () => {
     sectionReview: {
       eligibility: true,
     },
-    status: STATUS.IN_PROGRESS,
+    status: STATUS.ABANDONED,
```

Comment:
is there a reason for this to change to ABANDONED?

Reply:
Yes, `initialApplication.create` now has a default status of `IN_PROGRESS`.

`createAnApplication` accepts a status param that gets passed to `initialApplication.create`.

This test ensures that when a status is passed that is not the same as the default status, the application is created with the provided status.